plugins/woocommerce/src/Admin/API/OnboardingThemes.php

```diff
@@ -255,7 +255,8 @@ public function get_recommended_themes( $request ) {
 		$core_themes = array(
 			array(
 				'name'           => 'Twenty Twenty-Four',
-				'price'          => 'Free',
+				'price'          => __( 'free', 'woocommerce' ),
```

Comment:
I think we should capitalize `Free`, it looks much better imo. If you agree, please also apply to other instances 🙏 
```suggestion
				'price'          => __( 'Free', 'woocommerce' ),
```

Reply:
Agreed! Thanks for catching this. 🙏